packages/esm-generic-patient-widgets-app/src/obs-table/obs-table.component.tsx

```diff
@@ -66,12 +66,16 @@ const ObsTable: React.FC<ObsTableProps> = ({ patientUuid }) => {
             case 'Coded':
               rowData[obs.conceptUuid] = obs.valueCodeableConcept?.coding[0]?.display;
               break;
+
+            case 'DateTime':
+              rowData[obs.conceptUuid] = dayjs(new Date(obs.valueDateTime)).format(config.dateFormat);
```

Comment:
We have the `formatDate` function [here](https://github.com/openmrs/openmrs-esm-core/blob/95911641a40e15ccf9bb3a3a5298da7b14f4c002/packages/framework/esm-utils/src/omrs-dates.ts#L184) to format date as per the O3 designs. 
I guess we can use the `formatDate` here.

Reply:
This might also remove the use of formatting the datetime via configuration

Reply:
Thanks for the reply!

I made some changes and now I believe it's all according to the OpenMRS guidelines.

I still use the config variable but only to determine which formatter to use.

Reply:
Thanks a lot @icrc-agomes !